mews-api.md

```diff
@@ -917,6 +919,7 @@ There are certain rules that need to be followed in order for Mews to process th
 | `email` | `string` | _recommended_ | Email. |
 | `lastName` | `string` | required | Last name. |
 | `firstName` | `string` | optional | First name. |
+| `title` | `string` | optional | Accepted values: `mister`, `misses`, `miss` |
```

Comment:
Values are uppercase. + We extract this as separate enum check https://mews-systems.gitbook.io/connector-api/operations/reservations#reservation-state in connector API they mark it as string with types and linked to that type
e.g. https://mews-systems.gitbook.io/connector-api/operations/reservations#reservation-parameters reservation state
+ you are missing changelog